pegasus/src/curriculum_course.rb

```diff
@@ -94,7 +94,7 @@ def valid_lesson_directory?(lesson_id)
   end
 
   def valid_lesson_directory_name(lesson_dirname)
-    lesson_dirname != '.' && lesson_dirname != '..' && lesson_dirname.chars.first != '_'
+    lesson_dirname != '.' && lesson_dirname != '..' && lesson_dirname[0] != '_'
```

Comment:
hah!

Reply:
I do rather like our original implementation, but I get where Rubocop is coming from